store/store.go

```diff
@@ -253,16 +253,20 @@ func useStaging() bool {
 	return osutil.GetenvBool("SNAPPY_USE_STAGING_STORE")
 }
 
-func cpiURL() string {
+func apiURL() string {
 	// FIXME: this will become a store-url assertion
-	if u := os.Getenv("SNAPPY_FORCE_CPI_URL"); u != "" {
+	// backward-compatibility: this used to be "Click Package Index"
```

Comment:
we probably want a TODO: or XXX: marker here, I suppose we'll drop this at some point